Doc/library/traceback.rst

```diff
@@ -357,6 +357,7 @@ capture data for later printing in a lightweight fashion.
 
    .. versionchanged:: 3.11
       Added the *max_group_width* and *max_group_depth* parameters.
+      The exception's notes are now included in the output.
```

Comment:
Heh, while we're here, could we numerically sort the `.. versionchanged` notes attached to this function? We currently have a `.. versionchanged:: 3.13` note, then a `.. versionchanged:: 3.10` note, then a `.. versionchanged:: 3.11` note. I don't really mind whether they're sorted highest-to-lowest or lowest-to-highest, but the current situation is a little chaotic

Reply:
Actually it's worse than that. There are different indentations. I think some of them relate to the constructor.